modules/gdscript/gdscript_parser.cpp

```diff
@@ -3127,7 +3130,7 @@ GDScriptParser::ExpressionNode *GDScriptParser::parse_get_node(ExpressionNode *p
 			path_state = PATH_STATE_SLASH;
 		}
 
-		make_completion_context(COMPLETION_GET_NODE, get_node, context_argument++);
+		make_completion_context(COMPLETION_GET_NODE, get_node, context_argument++, true);
```

Comment:
What does this `true` (`p_force` parameter) do here? Does the first line above not fix the issue on its own?

Reply:
It is to ensure, that the first context gets overwritten, with the one which holds more information.
Currently this information (the Parser Node) is not used, as far as I know. But if someone wants to use it in the future it should get populated.
If we don't add the `true` we could delete the line entirely.

Reply:
Futureproofing in this case sounds good to me. I would suggest a comment but honestly the entire parser requires a few days to understand on its own, I don't think one would help sadly.

Reply:
Thinking about it again. I actually introduced a usage of the parser node in #85384 so this is important.